yarn-project/Earthfile

```diff
@@ -195,7 +194,17 @@ export-aztec-faucet:
 # We care about creating a slimmed down e2e image because we have to serialize it from earthly to docker for running.
 end-to-end-prod:
     BUILD ../spartan/+charts
-    FROM +cli-base
+    FROM +build
+    # A fast build is not enough for the end-to-end tests, we still need to run webpack
+    RUN cd end-to-end && yarn build:web
+    # Now that the web is compiled, we can trim down the image further
+    RUN rm -rf \
+        ../noir-projects \
+        ../l1-contracts \
+        ../barretenberg/ts/src \
+        ../barretenberg/ts/dest/node-cjs \
+        ../barretenberg/ts/dest/browser
```

Comment:
I'd wait for @ludamad to validate this change

Reply:
This was previously applied to `+cli-base`, but I cannot do it before building the web (need the bb ts artifacts before the cleanup). Now this step uses `+build` as base and performs this step manually